Research/Research/RSDTextInputTableItem.swift

```diff
@@ -100,10 +100,11 @@ open class RSDTextInputTableItem : RSDInputFieldTableItem {
     open func answerText(for answer: Any?) -> String? {
         if let text = pickerSource?.textAnswer(from: answer) ?? formatter?.string(for: answer) {
             return text
-        } else if let anyAnswer = answer {
+        } else if let anyAnswer = answer, !(anyAnswer is NSNull) {
```

Comment:
I'm confused--you're no longer having answer fall back to NSNull() in RSDInputFieldTableItemGroup but now you're checking for answer being NSNull() here?

Reply:
So yeah. The `NSNull` is a holdover from porting this from Obj-c and it causes problems where the value of `NSNull` was being returned as a text string. The goal was to remove the references to NSNull, but I missed some.

Reply:
ok